src/sentry/migrations/0294_backfill_alert_owners.py

```diff
@@ -0,0 +1,61 @@
+# Generated by Django 2.2.27 on 2022-04-08 22:24
+
+from django.db import migrations
+
+from sentry.new_migrations.migrations import CheckedMigration
+from sentry.utils.query import RangeQuerySetWrapperWithProgressBar
+
+
+def backfill_alert_owners(apps, schema_editor):
+    AlertRule = apps.get_model("sentry", "AlertRule")
+    OrganizationMember = apps.get_model("sentry", "OrganizationMember")
+    User = apps.get_model("sentry", "User")
+
+    for alert_rule in RangeQuerySetWrapperWithProgressBar(AlertRule.objects_with_snapshots.all()):
+        owner = alert_rule.owner
+        if not owner:
+            continue
+
+        valid_owner = False
+        if owner.type == 1:  # Actor is a User
+            user = User.objects.get(actor_id=owner.id)
+            if OrganizationMember.objects.filter(
+                organization_id=alert_rule.organization_id, id=user.id
+            ).exists():
+                valid_owner = True
```

Comment:
Won't this set all rules owned by a team to have no owner, regardless of org?

Reply:
Missed that, thanks.